plugins/k8s/cmd/scripts/bump-restart.sh

```diff
@@ -2,6 +2,8 @@
 
 set -e
 
+chroot /host /bin/bash /cedana/scripts/systemd-reset.sh
+
 # NOTE: This script assumes it's executed in the container environment
```

Comment:
This comment is for the script, can move it at the topmost. Good to merge after that.